iis-wf/iis-wf-affmatching/src/main/resources/eu/dnetlib/iis/wf/affmatching/model/MatchedAffiliation.avdl

```diff
@@ -8,9 +8,6 @@ protocol IIS {
         // eu.dnetlib.iis.metadataextraction.schemas.ExtractedDocumentMetadata#id
         string documentId;
         
-        // position in the document (from 1)
-        int position;
-        
```

Comment:
Sorry for this late comment but I just noticed it: since we have removed `position` field it does not represent relation between affiliation and organization anymore, does it? It is more relation between document and organization. Maybe we should rename it into `DocumentOrganization` or `MatchedOrganization` which will be accurate in both cases ;)